Directory.Build.props

```diff
@@ -102,6 +102,8 @@
     <SystemTextEncodingsWebVersion>8.0.0</SystemTextEncodingsWebVersion>
     <MicrosoftExtensionsConfigurationBinderVersion>8.0.0</MicrosoftExtensionsConfigurationBinderVersion>
     <MicrosoftExtensionsDependencyInjectionVersion>8.0.0</MicrosoftExtensionsDependencyInjectionVersion>
+    <MicrosoftGraphVersion>5.34.0</MicrosoftGraphVersion>
```

Comment:
we don't want this only for .net 8, so doesn't make sense to put it here. Where are the packages currently referenced? Just updated the versions there.